src/table/matrix_table.cpp

```diff
@@ -90,6 +90,19 @@ void MatrixWorkerTable<T>::Get(const std::vector<integer_t>& row_ids,
 }
 
 template <typename T>
+void MatrixWorkerTable<T>::Get(T* data, size_t size, integer_t* row_ids,
```

Comment:
@feiga This function is implemented by me by referring to the one above. In that function the `size`   is checked with `num_col_` [1].

[1] https://github.com/Microsoft/multiverso/pull/62/files/225562384d692162659131496f99e15ac2b1e0c9#diff-fa4535afda5d4316b5601c83a8de7cc0R82